project_event/report/project_task_activity_report.xml

```diff
@@ -43,37 +39,30 @@
       <div style="border-bottom: 2px solid black">
         <table class="table">
           <tr>
-            <td> <t t-esc="activity_label"/> <t t-esc="name"/>  </td>
+            <td>Activity: <t t-esc="name"/>  </td>
           </tr>
           <tr>
-            <td width="50%"> <t t-esc="date_label"/>
-              <t t-esc="datetime.datetime.strptime(start, '%Y-%m-%d %H:%M:%S').strftime('%Y-%m-%d')"/>
+            <td width="50%">Date: <t t-esc="datetime.datetime.strptime(start, '%Y-%m-%d %H:%M:%S').strftime('%Y-%m-%d')"/>
             </td>
-            <td width="50%"> <t t-esc="number_spectators_label"/></td>
+            <td width="50%">Number of spectators: </td>
             </tr>
             <tr>
-              <td> 
-                <t t-esc="start_hour_label"/>
-                <t t-esc="datetime.datetime.strptime(start, '%Y-%m-%d %H:%M:%S').strftime('%H:%M')"/>              
+              <td>Hour: <t t-esc="datetime.datetime.strptime(start, '%Y-%m-%d %H:%M:%S').strftime('%H:%M')"/>              
```

Comment:
It didn't take the summer hour

Reply:
I do not see this problem

Reply:
The TZ that you chose on user preferences should be the one you see